grails-app/services/au/org/ala/collectory/EmlImportService.groovy

```diff
@@ -20,18 +20,18 @@ class EmlImportService {
         guid:  { eml -> eml.@packageId.toString() },
         pubDescription: { eml -> this.collectParas(eml.dataset.abstract?.para) },
         name: { eml -> eml.dataset.title.toString() },
-        email: { eml ->  eml.dataset.contact?.electronicMailAddress?.text() },
+        email: { eml ->  eml.dataset.contact[0]?.electronicMailAddress?.text() },
```

Comment:
@vjrj Im wondering what happens for datasets with zero contacts....

Reply:
@djtfmartin I synced all our ipt DRs in empty testing collectory successfully but I need to check if we have some dr without contacts. Let me check.

Reply:
@djtfmartin we were talking here and do some extra tests and the IPT does not allows to publish DRs without contacts.

Reply:
For the record: https://eml.ecoinformatics.org/images/eml-dataset.png

![2021-11-20-11-10-04-screenshot](https://user-images.githubusercontent.com/180085/126294602-a652e7ca-f56c-4543-80f5-91b20957e687.png)

Reply:
Hmmm interesting. So its not a valid EML without a contact...